ssl/s3_enc.c

```diff
@@ -412,16 +412,17 @@ int ssl3_digest_cached_records(SSL *s, int keep)
 }
 
 void ssl3_digest_master_key_set_params(const SSL_SESSION *session,
```

Comment:
Regarding this function, please consider #9126

Reply:
@levitte  I have merged your changes into this PR (since I needed to test that the ssl3 code actually works).